Tests/EventListener/RouteAnnotationEventListenerTest.php

```diff
@@ -34,7 +34,7 @@ public function testNoAnnotation()
      */
     public function testInvalidSitemapFalse()
     {
-        $this->setExpectedException('InvalidArgumentException');
+        $this->setExpectedException('\InvalidArgumentException');
```

Comment:
I propose to unify all usages of ```InvalidArgumentException``` in the file. 
We can use ```InvalidArgumentException``` for each usage(without backslash). 
@Koc What do you think?